samples/matter/lock/configuration/nrf5340dk_nrf5340_cpuapp/pm_static_dfu.yml

```diff
@@ -45,9 +45,49 @@ mcuboot_secondary_1:
     size: 0x40000
     device: MX25R64
     region: external_flash
-external_flash:
+app_1_download_app:
     address: 0x133000
-    size: 0x6CD000
+    size: 0xf3000
+    device: MX25R64
+    region: external_flash
+app_1_download_net:
+    address: 0x226000
+    size: 0x40000
+    device: MX25R64
+    region: external_flash
+app_2_download_app:
+    address: 0x266000
+    size: 0xf3000
+    device: MX25R64
+    region: external_flash
+app_2_download_net:
+    address: 0x359000
+    size: 0x40000
+    device: MX25R64
+    region: external_flash
+app_1_app:
```

Comment:
Just a thought.. what about to call it `image_1_app_core` and `image_1_net_core`.. eventually skip `_core`.

Reply:
> I hope that the maintenance cost of the another pm_static file would not be that hard.

It's always a pain that if we don't want to create another prj.conf (and I assume we don't), the pm_static will not be selected automatically. We will have to set it in CMakeLists before Zephyr env is loaded, so we cannot use simple Kconfig to decide about it, but some other cmake flag.

Reply:
The thing is that `image` is used in so many contexts that it's hard to know what it means. Multi-image typically means that we have app and net core images. So I would prefer we craft a new term to avoid confusion. I proposed "variant" earlier but I noticed it wasn't accepted in the team, so perhaps we need to discuss that :)

Reply:
F2F: Let's use `app_1_core_net` and `app_1_core_app` names.